data_structures/avltree_map.cpp

```diff
@@ -0,0 +1,226 @@
+/**
+ * @file avltree_map.cpp
```

Comment:
Best not to include the name, otherwise, you will have to set the full directory for Doxygen to detect it.
```suggestion
 * @efugier 
```